src/api/users/src/routes/users.js

```diff
@@ -140,6 +142,43 @@ router.put(
     const { id } = req.params;
     const { body } = req;
 
+    try {
+      const userRef = db.doc(id);
+      const doc = await userRef.get();
+
+      if (!doc.exists) {
```

Comment:
This code seems to be duplicated between a few `put` calls now.  Can you extract it into middleware and share it?

Reply:
Duplication removed, code moved to `updateUser` in `schema.js`